code/modules/mob/living/carbon/xenomorph/castes/hivemind/abilities_hivemind.dm

```diff
@@ -5,3 +5,26 @@
 
 /datum/action/xeno_action/return_to_core/action_activate()
 	SEND_SIGNAL(owner, COMSIG_XENOMORPH_CORE_RETURN)
+
+/datum/action/xeno_action/activable/nightfall
+	name = "Nightfall"
+	action_icon_state = "nightfall"
+	mechanics_text = "Shut down all light for 10 seconds"
```

Comment:
mechanics_text = "Shut down all light for 5 seconds


I think its a cool ability and can not wait for that oh shit moment when my lights die.